features/support/env.rb

```diff
@@ -89,19 +89,40 @@ def debrand_string(str)
 case browser
 when :phantomjs
   require 'capybara/poltergeist'
-  Capybara.register_driver :poltergeist do |app|
-    Capybara::Poltergeist::Driver.new(app,
+   Capybara.register_driver :poltergeist do |app|
+    driver = Capybara::Poltergeist::Driver.new(app,
```

Comment:
apparently a small bug

you should set "driver" variables in all code paths (chrome, firefox, phantomjs)

ie you should do 
    driver =
in all three case

great pr, by the way...